src/eth/TransactionManager.js

```diff
@@ -57,7 +57,20 @@ export default class TransactionManager extends PublicService {
       (async () => {
         // so we do our async operations inside this immediately-executed
         // async function.
-        const txOptions = await this._buildTransactionOptions(options);
+        let data = contract.interface.functions[method](...args).data;
+        if (options.dsProxy) {
+          const proxyAddress = await this.get('proxy').currentProxy();
+          const proxy = this.get('proxy')._getUnwrappedProxyContract(
```

Comment:
if `_getUnwrappedProxyContract` is now expected to be used outside the service it shouldn't start with `_`